test/cpp/ext/otel/BUILD

```diff
@@ -61,7 +61,64 @@ grpc_cc_test(
         ":otel_test_library",
         "//:grpc++",
         "//src/cpp/ext/otel:otel_plugin",
+        "//test/core/promise:test_context",
+        "//test/core/test_util:grpc_test_util",
+    ],
+)
+
+grpc_cc_test(
+    name = "grpc_text_map_carrier_test",
+    srcs = [
+        "grpc_text_map_carrier_test.cc",
+    ],
+    external_deps = [
+        "gtest",
+        "otel/api",
+    ],
+    tags = [],
+    deps = [
+        "//:grpc++",
+        "//src/cpp/ext/otel:otel_plugin",
+        "//test/core/promise:test_context",
+        "//test/core/test_util:grpc_test_util",
+    ],
+)
+
+grpc_cc_test(
+    name = "grpc_trace_bin_text_map_propagator_test",
+    srcs = [
+        "grpc_trace_bin_text_map_propagator_test.cc",
+    ],
+    external_deps = [
+        "gtest",
+        "otel/api",
+    ],
+    tags = [],
+    deps = [
+        "//:grpc++",
+        "//src/cpp/ext/otel:otel_plugin",
+        "//test/core/promise:test_context",
+        "//test/core/test_util:grpc_test_util",
+    ],
+)
+
+grpc_cc_test(
+    name = "otel_tracing_test",
+    srcs = [
+        "otel_tracing_test.cc",
+    ],
+    external_deps = [
+        "gtest",
+        "otel/api",
+        "otel/exporters/memory:in_memory_span_exporter",
+        "otel/sdk/src/trace",
+    ],
+    tags = [],
```

Comment:
Please add the "grpc:broken-internally" tag here, so that this doesn't race with Craig's cl/731158267.